src/System.Text.Primitives/System/Text/Formatting/PrimitiveFormatter.cs

```diff
@@ -15,39 +15,39 @@ namespace System.Text
     public static partial class PrimitiveFormatter
     {
         #region Integers
-        public static bool TryFormat(this byte value, Span<byte> buffer, out int bytesWritten, TextFormat format, EncodingData encoding)
+        public static bool TryFormat(this byte value, Span<byte> buffer, out int bytesWritten, TextFormat format = default(TextFormat), EncodingData encoding = default(EncodingData))
         {
             return IntegerFormatter.TryFormatUInt64(value, 1, buffer, out bytesWritten, format, encoding);
         }
 
-        public static bool TryFormat(this sbyte value, Span<byte> buffer, out int bytesWritten, TextFormat format, EncodingData encoding)
+        public static bool TryFormat(this sbyte value, Span<byte> buffer, out int bytesWritten, TextFormat format = default(TextFormat), EncodingData encoding = default(EncodingData))
         {
```

Comment:
This is the official .NET guideline, but it means the parameters could not be defaulted. The guidelines were developed before we allowed default parameters in public APIs, and now these two guidelines (ok to use default and out parameters being last) seem not compatible with each other.

@terrajobst, we should talk about the guidelines at some point and see if we need to tweak them. But for now, I think this change is ok, and will give us more experience when we talk about the guidelines and how to adjust them.